modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -109,6 +109,12 @@ All three basic types of transports are supported in clusters in different ways,
 ** Note that, in a cluster, VM transport queues are a shared, cluster-wide resource. The cluster will automatically synchronize access to the VM transport queues. Because of this, a message written to a VM queue can be processed by any cluster node. This makes VM ideal for sharing work among cluster nodes.
 * Resource-based
 ** Mule HA Clustering automatically coordinates access to each resource, ensuring that only one clustered instance accesses each resource at a time. Because of this, it is generally a good idea to immediately write messages read from a resource-based transport to VM queues. This allows the other cluster nodes to take part in processing the messages.
++
+[NOTE]
+====
+Have in mind that Mule only coordinates that different nodes in the cluster don’t try to access the same resource.
```

Comment:
I don't think you need the first sentence in the Note since it is explained above the note. Suggested rewrite of the second sentence.:

If you are running Mule applications or non-Mule applications outside the cluster that need to access resources within the cluster, you must implement logic to lock those resources.

Reply:
Thanks, I agree that the first sentence is redundant. Regarding the second, I'm not sure if "resources within the cluster" is accurate because I think the resources could be located outside the cluster as well. 
The context that needs to be covered here is when: _non-Mule apps or Mule apps outside the cluster need to access the same resources as the nodes within a cluster_

Reply:
That works.

Reply:
I thought you might want to put Mule apps outside the cluster before non-Mule apps. Whatever is most common I guess.

Reply:
I ordered it that way because non-Mule apps will always be outside of the cluster, while Mule apps can be in or out, that's why the clarification affects directly Mule apps.